tests/cpp/common/test_json.cc

```diff
@@ -432,4 +524,35 @@ TEST(Json, IntVSFloat) {
     ASSERT_EQ(ptr, 2503595760);
   }
 }
+
+TEST(Json, RoundTrip) {
+  uint32_t i = std::numeric_limits<uint32_t>::max() / 100 * 0;
```

Comment:
What is happening here? A comment on what this test is doing would help me.

Reply:
Removed.  Previously I placed a progress bar while running brute force test.  The upstreamed test is not full coverage in order to reduce unittest time down to 1 sec.